src/Stoa.ts

```diff
@@ -21,10 +33,45 @@ class Stoa {
             res.send(req.params.address + ':' + req.query.height);
         });
 
+        /**
+         * When a request is received through the `/push` handler
+         * JSON block data is parsed and stored on each storage.
+         */
         this.stoa.post("/push",
             (req: express.Request, res: express.Response, next: express.NextFunction) => {
-            console.log(req.body);
-            res.status(200).send();
+
+            var block: any;
+            if (req.body.block == undefined)
+            {
+                res.status(400).send("Missing 'block' object in body");
+                return;
+            }
+
+            try {
+                block = JSON.parse(req.body.block);
+            } catch(e) {
+                res.status(400).send("Not a valid JSON format");
+                return;
+            }
+
+            console.log(block);
+
+            blockStorage.put(block, (err: any) =>
+            {
+                blockStorage.close;
+                if (err != null)
+                {
+                    console.error(err);
```

Comment:
This should come with a message so it can be grepped for

Reply:
? It returns an error message during storage.

Reply:
It returns an unpredictable error message.
What I'm suggesting is:
```suggestion
                    console.error("Failed to store the payload of a push to the DB: " + err);
```

This way someone can grep for "Failed to store the payload of a push to the DB" and know where the error is logged. because `console.log(err)` is not something you can look for easily.